source/server-selection/server-selection.rst

```diff
@@ -341,6 +341,14 @@ smallestMaxStalenessSeconds
 A constant, 90 seconds. See "Smallest allowed value for maxStalenessSeconds"
 in the Max Staleness Spec.
 
+serverSelector
+~~~~~~~~~~~~~~
+
+An optional, application-provided function that augments the server selection
```

Comment:
What I mean by "optional" here is that it's not required for the application to provide one.  It doesn't say whether implementers are required to do this and we had agreed elsewhere that this should be a MAY.

I could reword to:

     Implementations MAY allow configuration of an optional, application-provided function ...

but then do I need to add that language in the sections below as well, or is it enough just to do it here?

Reply:
I think just adding the "MAY" language here is clear enough.

Reply:
Done